app/javascript/components/close_contact/CloseContact.js

```diff
@@ -265,6 +311,11 @@ const schema = yup.object().shape({
     .email('Please enter a valid email.')
     .max(200, 'Max length exceeded, please limit to 200 characters.')
     .nullable(),
+  last_date_of_exposure: yup.date('Date must correspond to the "mm/dd/yyyy" format.').nullable(),
+  assigned_user: yup
```

Comment:
I think this validation can just be done the same way its done in other places within the app in that we just make it impossible for the user to even type anything other than a number between 1-9999 and this can be done by checking for that on `handleChange`

Reply:
Resolved within `yup` using `typeError`.

Reply:
I think we should try to keep the assigned user field as consistent as possible with all the other instances of it in the app instead of relying on yup, this should behave similarly if not exactly the same way as the assigned user input field from the enrollment/exposure component, and I think you can just copy/paste that code

Reply:
I had to add some props, but I did that. I also added additional yup validation. So the behavior is the same as was, but I dont see a problem with additional yup validation